content/boilerplates/notes/0.8.md

```diff
@@ -5,7 +5,7 @@
 deployment [use-cases](/docs/tasks/traffic-management/). For folks upgrading from earlier releases there is a
 [migration guide](/docs/setup/kubernetes/upgrade/) and a conversion tool built into `istioctl` to help convert your configuration from the old model.
 
-- **Streaming Envoy configuration**. By default Pilot now streams configuration to Envoy using its [ADS API](https://github.com/envoyproxy/data-plane-api/blob/master/XDS_PROTOCOL.md). This new approach increases effective scalability, reduces rollout delay and should eliminate spurious 404 errors.
+- **Streaming Envoy configuration**. By default Pilot now streams configuration to Envoy using its [ADS API](https://github.com/envoyproxy/data-plane-api/blob/master/xds_protocol.rst). This new approach increases effective scalability, reduces rollout delay and should eliminate spurious 404 errors.
```

Comment:
This change and the corresponding Chinese change have already been introduced to the source base this morning. So if you rebase on the latest bits, you won't need these two changes in your commit.